parastell/parastell.py

```diff
@@ -574,6 +529,63 @@ def check_inputs(
             )
             logger.error(e.args[0])
             raise e
+        
+
+def build_full_model(stellarator, invessel_build, magnet_coils, source_mesh):
+    """Use all of the input to make a complete stellarator model with in-vessel
+    components, magnets, and a source mesh.
+
+    Arguments:
+        stellarator (object): Stellarator class object.
+        invessel_build (dict): dictionary of RadialBuild and InVesselBuild
+            parameters.
+        magnet_coils (dict): dictionary of MagnetSet parameters.
+        source_mesh (dict): dictionary of SourceMesh parameters.
+    """
+    stellarator.construct_invessel_build(**invessel_build)
+    stellarator.construct_magnets(**magnet_coils)
+    stellarator.construct_source_mesh(**source_mesh)
```

Comment:
@connoramoreno  These all have positional arguments - shouldn't we call them with those arguments extracted from the dictionaries explicitly here?

Reply:
Defining the positional arguments and unpacking the respective dictionaries in the same function call defines those positional arguments twice, causing a `TypeError`

Reply:
Just unpacking the dictionaries will appropriately set the positional arguments.

Reply:
If we want to explicitly set positional arguments, we'll need to do something similar to what I had before where secondary dictionaries are created for the `kwargs`

Reply:
OK - Is there an error if one of the positional arguments is missing?  (Maybe we should add a test for that at some point)

Reply:
E.g., I removed `poloidal_angles` from `config.yaml` to test:
```
TypeError: Stellarator.construct_invessel_build() missing 1 required positional argument: 'poloidal_angles'
```

Reply:
I think this is ready to merge, then...

Reply:
Agreed!